src/fsharp/FSComp.txt

```diff
@@ -1435,8 +1435,9 @@ notAFunctionButMaybeDeclaration,"This value is not a function and cannot be appl
 3230,chkNoWriteToLimitedSpan,"This value can't be assigned because the target '%s' may refer to non-stack-local memory, while the expression being assigned is assessed to potentially refer to stack-local memory. This is to help prevent pointers to stack-bound memory escaping their scope."
 3231,tastValueMustBeLocal,"A value defined in a module must be mutable in order to take its address, e.g. 'let mutable x = ...'"
 3232,tcIsReadOnlyNotStruct,"A type annotated with IsReadOnly must also be a struct. Consider adding the [<Struct>] attribute to the type."
-3233,chkStructsMayNotReturnAddressesOfContents,"Struct members cannot return the address of fields of the struct by reference"
```

Comment:
Hmm, I think these numbers should be returned to their original value.  I have no idea how this happened I guess it was some merge nonsense.

Reply:
Yes I should, otherwise this basically undoes this change:  https://github.com/Microsoft/visualfsharp/commit/d97c7976ccbd0a2605fcb9db055ead82b8049598